packages/plugins/auto-doc/src/pages/template-overview/template-overview.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts" context="module">
+
+    export type Template = {
+        name: string,
+        description: string,
+        lastEdited: Date,       
+    }
+</script>
+
+<script lang="ts">
+    import {Table} from '@/components'
+
+    let allTemplates : Template[] = [
+        {
+            name: 'Template1',
+            description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda, asperiores accusamus quis sapiente sequi facilis?',
+            lastEdited: new Date("2024-10-15T08:38:00"),
+        },
+        {
+            name: 'Template256',
+            description: 'A simple description',
+            lastEdited: new Date("2024-10-10T13:24:00")
+        },
+        {
+            name: 'CYK_Template',
+            description: 'I have an invalid Date ',
+            lastEdited: new Date("foo"),
+        },
+        {
+            name: 'I surely have quite a long title name.',
+            description: "Lorem ipsum dolor sit amet consectetur.",
+            lastEdited: new Date("2024-10-20T10:55:32"),
+        },
+        {
+            name: 'AUOBHBT Template',
+            description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta placeat itaque minus praesentium sequi deleniti voluptatibus dicta!',
+            lastEdited: new Date(),
+        },
+    ]    
+</script>
+
+<main class="tamplate-overview">
+    <header class="template-controls">
+        <button class="btn-pill btn-pill-primary"> + Add template</button>
+        <button class="btn-pill btn-pill-outlined">Export Documents</button>
+        <button class="btn-pill btn-pill-outlined">Filter</button>
```

Comment:
On the other hand, I see that using
`<Button color="secondary" variant="unelevated"><Label>Some Label</Label></Button>`
Does not work as expected : color is still some kind of primary darker color...
Should we create a wrapper with custom class (to also add more custom colors), or maybe try to fix it ?
I don't know if it comes from the `ui > theme` setup or not.

SMUI is based on [material-components-web](https://github.com/material-components/material-components-web) and the button component itself is build based [on the material component theme ](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme).

We [override the stock colors here](https://github.com/sprinteins/oscd-plugins/blob/main/packages/ui/src/assets/styles/theme.css) as per of the [SMUI documentation](https://sveltematerialui.com/THEMING.md).

This warning might be the key : `Note that not all of the colors in SMUI read from CSS variables, so some parts will still use the colors defined in the original Sass compile.`
We might need to rerender all styles with our customs style.

I don't know what is the better option for us, maybe that's why you use custom buttons in the end ! 🙃

Reply:
Oh right, they were supposed to be some type of placeholders at the beginning.
It makes sense to use the smui Button since it is the one we're using throughout the application. 
I couldn't replicate the issue you're having with the color @michelguerin. Maybe we can address it on a call